src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -1,10 +1,24 @@
 import React, { Children } from 'react';
+import PropTypes from 'prop-types';
 import styled, { css } from 'react-emotion';
 
 import Button from '../Button';
 import { childrenPropType } from '../../util/shared-prop-types';
+import { isEqual } from '../../util/fp';
 
-const listStyles = ({ theme }) => css`
+const LEFT = 'left';
+const RIGHT = 'right';
+const ALIGMENT_TYPES = [LEFT, RIGHT];
+
+const isRight = isEqual(RIGHT);
+
+const alignmentStyles = align =>
+  isRight(align) &&
+  css`
```

Comment:
How would this work with our static style extraction? 

I think it's better to separate the `alignmentStyles` from the `baseStyles` and add a label:

```js
const alignmentStyles = align =>
  isRight(align) &&
  css`
    label: button-group--left;
    justify-content: flex-end;
  `;
```